dataservices/service/api/standard.go

```diff
@@ -23,24 +21,20 @@ import (
 	"github.com/tidepool-org/platform/dataservices/service/context"
 	"github.com/tidepool-org/platform/environment"
 	"github.com/tidepool-org/platform/log"
-	"github.com/tidepool-org/platform/service/middleware"
+	"github.com/tidepool-org/platform/service/api"
 	"github.com/tidepool-org/platform/userservices/client"
 	"github.com/tidepool-org/platform/version"
 )
 
 type Standard struct {
-	versionReporter         version.Reporter
-	environmentReporter     environment.Reporter
-	logger                  log.Logger
+	*api.Standard
 	userServicesClient      client.Client
 	dataFactory             data.Factory
 	dataStore               store.Store
 	dataDeduplicatorFactory deduplicator.Factory
-	api                     *rest.Api
-	statusMiddleware        *rest.StatusMiddleware
 }
 
-func NewStandard(versionReporter version.Reporter, environmentReporter environment.Reporter, logger log.Logger, userServicesClient client.Client, dataFactory data.Factory, dataStore store.Store, dataDeduplicatorFactory deduplicator.Factory, routes []service.Route) (*Standard, error) {
+func NewStandard(versionReporter version.Reporter, environmentReporter environment.Reporter, logger log.Logger, userServicesClient client.Client, dataFactory data.Factory, dataStore store.Store, dataDeduplicatorFactory deduplicator.Factory) (*Standard, error) {
```

Comment:
I don't think we need to adjust now but just wondering if it would be easier for these to list the params on a new line for readability?